src/app/SanctionedCountryErrorScreen.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+import fontStyles from 'src/styles/fonts'
+
+class SanctionedCountryErrorScreen extends React.Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <Text style={styles.errorMessage}>{'Sorry, Valora is not support in your location'}</Text>
+      </View>
+    )
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    flex: 1,
```

Comment:
we should add some padding so that the text doesn't touch the screen sides for small screens

Reply:
I added padding!